crates/vm/levm/src/call_frame.rs

```diff
@@ -11,6 +11,9 @@ use ethrex_common::{Address, U256};
 use ethrex_common::{H256, types::Code};
 use std::{collections::HashMap, fmt};
 
+// u64's that make up an U256
+const U64_PER_U256: usize = 256 / 64;
```

Comment:
I'm nitpicking, but I think this is safer:

```suggestion
/// [`u64`]s that make up a [`U256`]
const U64_PER_U256: usize = U256::MAX.0.len();
```